tests/monitoring/prometheus_utils.go

```diff
@@ -70,33 +70,36 @@ func waitForMetricValue(client kubecli.KubevirtClient, metric string, expectedVa
 
 func waitForMetricValueWithLabels(client kubecli.KubevirtClient, metric string, expectedValue int64, labels map[string]string) {
 	EventuallyWithOffset(1, func() int {
-		v, err := getMetricValueWithLabels(client, metric, labels)
+		i, err := getMetricValueWithLabels(client, metric, labels)
 		if err != nil {
 			return -1
 		}
-		i, err := strconv.Atoi(v)
-		Expect(err).ToNot(HaveOccurred())
 		return i
 	}, 3*time.Minute, 1*time.Second).Should(BeNumerically("==", expectedValue))
 }
 
-func getMetricValueWithLabels(cli kubecli.KubevirtClient, query string, labels map[string]string) (string, error) {
+func getMetricValueWithLabels(cli kubecli.KubevirtClient, query string, labels map[string]string) (int, error) {
```

Comment:
@machadovilaca For a follow-up PR I think we can use named returns with go defaults here, instead of using custom defaults or "magic" numbers